tests/test_raid.py

```diff
@@ -1,10 +1,13 @@
 import json
 import mock
 import pytest
+import subprocess
 import XenAPIPlugin
 
 from raid import check_raid_pool
 
+COMMAND = ['mdadm', '--detail', '/dev/md127']
```

Comment:
Could you use a more explicit variable name? Not very clear and also if other tests are added it will not help regarding the maintenance.